src/VscPreviewExtension/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "DfmPreview",
   "displayName": "DfmPreview",
-  "description": "Preview the DocFX Flavored Markdown and token tree in Visual Studio Code",
+  "description": "Preview DocFX generated Html and token tree in Visual Studio Code",
```

Comment:
token tree is not generated by DocFX?

---
In reply to: [108832032](https://github.com/dotnet/docfx/pull/1507#discussion_r108832032) [](ancestors = 108832032)